app/src/main/java/com/example/mystore/ui/products/ProductDetailsScreen.kt

```diff
@@ -138,70 +121,64 @@ fun ProductDetailsScreen() {
                 Row(
                     modifier = Modifier
                         .fillMaxWidth()
-                        .padding(top = 20.dp),
+                        .padding(top = dimensionResource(id = R.dimen.padding_top_medium_20)),
                     horizontalArrangement = Arrangement.SpaceBetween
                 ) {
                     Text(
                         text = it.title,
-                        fontWeight = FontWeight.Bold,
-                        fontSize = MaterialTheme.typography.labelLarge.fontSize,
-                        modifier = Modifier.width(200.dp)
+                        modifier = Modifier.width(dimensionResource(id = R.dimen.width_200)),
+                        style = MaterialTheme.typography.bodyLarge.copy(
+                            fontWeight = FontWeight.Bold
+                        )
                     )
                     RatingBar(
                         rating = it.rating.rate,
-                        stars = 5
+                        stars = Constants.MAX_RATING
                     )
                 }
                 Text(
-                    modifier = Modifier.padding(bottom = 20.dp),
+                    modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.padding_bottom_medium)),
                     text = "Category: ${it.category}",
-                    color = Color.Gray,
-                    fontSize = MaterialTheme.typography.labelMedium.fontSize
+                    color = Gray,
+                    style = MaterialTheme.typography.labelSmall
                 )
                 Text(
-                    modifier = Modifier.padding(bottom = 20.dp),
-                    text = it.description
+                    modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.padding_bottom_medium)),
+                    text = it.description,
+                    style = MaterialTheme.typography.bodyLarge
                 )
                 Text(
-                    modifier = Modifier.padding(bottom = 20.dp),
+                    modifier = Modifier.padding(bottom = dimensionResource(id = R.dimen.padding_bottom_medium)),
                     text = "$${it.price}",
-                    fontWeight = FontWeight.Bold,
-                    fontSize = MaterialTheme.typography.titleLarge.fontSize
+                    style = MaterialTheme.typography.displayMedium.copy(
+                        fontWeight = FontWeight.Bold
+                    )
                 )
                 Button(
```

Comment:
This button can be reused alongside the other button as a shared reusable composable, parametrized in a way that can satisfy both scenarios.